.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
Not an editor command: Lexplore. Something like that. i changed to "Explore" and it works, however worked as "vsp"